Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.mm

```diff
@@ -708,7 +709,7 @@ - (BOOL)prefersStatusBarHidden
         ALWAYS_LOG(LOGIDENTIFIER, model->logIdentifier(), ", Creating AVPlayerLayerView");
         auto playerView = adoptNS([allocWebAVPlayerLayerViewInstance() init]);
 
-        auto *playerLayer = (WebAVPlayerLayer *)[playerView layer];
+        auto playerLayer = RetainPtr { (WebAVPlayerLayer *)[playerView layer] };
```

Comment:
I think we usually do:
```
RetainPtr playerLayer { (WebAVPlayerLayer *)[playerView layer] };
```

Reply:
Fixed.  (Sorry, you keep flagging this on my patches. I will double-down on remembering to use this format.)